lib/utils/queryable.js

```diff
@@ -10,6 +10,7 @@ const cleanLeadingDot = str =>
 const parseKeys = (key) => {
   const sqBracketItems = new Set()
   const parseSqBrackets = (str) => {
+    str = str.replace('[]', '[-0]')
```

Comment:
what if i have an object with a literal key of "-0"?

Reply:
It seems just like using a string placeholder won’t work, because any string could be an object key - ie, it’s an implementation question, not a semantics question.

Reply:
For an array, negative indexes could indeed be supported - but you’d have to traverse down to know that’s what it was.